src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2.java

```diff
@@ -90,6 +91,7 @@
         oneLineSummary = "Call somatic SNPs and indels via local re-assembly of haplotypes",
         programGroup = VariantProgramGroup.class
 )
+@DocumentedFeature
```

Comment:
In the current doc system, you need to use the DocumentedFeature annotation to get a tool to show up in doc. We do also have the ability to use embedded (custom) javadoc tags - the doc system will populate the freemarker template properties with these, and you can use them in html however you like. However, the command line help parser doesn't look at the javadoc, so that doesn't help with anything that needs to surface in command line help. There is a [Barclay PR](https://github.com/broadinstitute/barclay/pull/51) that implements an annotation that is integrated with both the CLP and the doc system to tag "experimental" tools (its actually called "EvaluationOnly" in the PR - but we can change that). It causes a property to be emitted to the freemarker template, and fixed string to be embedded in the command line output. Of course, any of it can be changed...